tg_bot.py

```diff
@@ -40,9 +41,15 @@ def display_tweet(self, context):
         print('starting display parameter')
         job = context.job
         global KILL_SIGNAL
+        previous_date =  datetime.now() + timedelta(days=-1)
         tid = pd.read_csv(self.dirpath + 'tid_store.csv')
-        for index, tid_store in tid.iterrows():
-            
+        tid["Date_time"]= pd.to_datetime(tid["Date_time"])
+        
+        for index, tid_store in tid[::-1].iterrows():
+            scrape_date = tid_store['Date_time']
+            if scrape_date<=previous_date:
```

Comment:
Also would this collect the first element of the group of previous date?

That is, assuming now is 15/05 and you are requesting for 14/05, are you taking the first occurrence of 14/05 from the bottom or the last occurrence of it in the tid store?

Reply:
I believe it takes the last occurrence of 15/05(ie the latest tweet id), from bottom to top till it reaches 14/05

Reply:
yes what @ananthanandanan said is right

Reply:
I didn't understand

Reply:
```Date_time,Id
2020-05-15 07:33:49.177777,1261286201107120129
2020-05-15 07:33:52.177777,1261296000716738560
2020-05-16 23:07:46.605533,1261710344709423110```
Now if this is the tid_store.csv, it will only send the last shown tweet.

Reply:
and it moves from the end of the file date to `the previous_date`